flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -129,6 +130,12 @@
 	 */
 	protected final TypeSerializer<W> windowSerializer;
 
+	/**
+	 * The allowed lateness for elements.
```

Comment:
This needs to be updated. The allowed lateness is used both for dropping of late elements (based on the window into which they fall) and garbage collection of window state). The default is not set in this operator but in `WindowedStream`.